README.md

```diff
@@ -6,11 +6,66 @@ Automation test repository before code is integrated into [fasttrack repository]
 
 > All reminders are defined in a template file stored in the repo (markdown)
 
+## Questions for the team
+- How much data should we embed in Project board custom field and how much should be in the issues itself...
+  - Start and End Dates (Title and fields for now)
+  - Follow-up date triggers (Field)
+  - OKR information (?)
+  - Organizations (?)
+  - Platform type (Field)
+  - Other?
+
+## Process Overview
+1. TRIAGE (No status on the project board)
+    - **Trigger:** Request is opened, filled out by AM/CSM/CSA/SE/TSAM, automatically labeled `Triage` and assigned to FastTrack Ops (dmckinstry) via the `ISSUE_TEMPLATE`
+    - Workflow moves the issue onto the project (no status), sets the FollowDate to tomorrow, and appends the checklist to the description
+    - Requester and FastTrack Ops iterate on preapproval process (outside of any automation)
+    - FastTrack Ops presents to customer and documents the results
+2. REJECTED (Closed)
+    - **Trigger:** FastTrack Ops issues a "/REJECT" command; this can be at any time during the entire engagement process but will typically be before or during the FastTrack Kickoff call (first call) with the customer
+    - Issue is removed from the project board and closed
+    - @mention is appended to the issue discussion via automation notifying the person who opened the request and @dmckinstry (e.g., "*:point_up: @someone @dmckinstry - This request has been rejected per tha above comments. Please let us know if you believe this was a mistake.*")
+3. APPROVED (1-Approved)
+    - **Trigger:** FastTrack Ops approves via the "/APPROVE" command
+    - Workflow removes the `Triage` label and moves the issue to the `1-Approved` status in the project board
+      - @tspascoal :point_right: I do not anticipate the need to auto-assign as part of the approval process. It is possible that we will know the right assignee but that usually follows looking at the schedule.  And if we do know, I am concerned about too much clutter for the DevOps Architect before things settle... Your thoughts?
+3. SCHEDULED (2-Scheduled)
+    - **Trigger:** FastTrack Ops formats the title into standard format including dates, assignsthe  DevOps Architect(s) and manually moves to the `2-Scheduled` status on the project board
+4. DELIVERING (3-Delivering)
+    - **Trigger:** Workflow moves the request into the `3-Delivering` column the morning of the start date.
```

Comment:
```suggestion
    - **Trigger:** Workflow:
      -  moves the request into the `3-Delivering` column the morning of the start date.
      - Adds the placeholder for the trip report (currently done when issue gets an `confirmed` label)
      - Adds the checklist to the issue (see reason on comment above)
```

Reply:
Are you talking about adding the DELIVERY checklist when it moves to 3-Deliverying?  IF so, we can try it.  Most of the checklist is in the prep stage and I need it for pre-approval and scheduled - well before delivery.  Today, I fill in what I can in the delivery checklist during the pre-delivery process.  I don't need to do that and can leave it to the architect.  But if I do know that it is GHEC-EMU and/or the orgs and/or...  then I figure it makes sense to capture it when I know it instead of waiting for the architect to re-discover and record it.

Reply:
Ok. I will break the checklist into two then.....

The pieces are all there, just need to call them in two different places with different parameters.

Would that work?